SVA/lib/handlers.S

```diff
@@ -507,24 +507,58 @@ SVAsyscall:
    * if this is the fork(), vfork(), rfork(), or pdfork() system calls.  The
    * system call number will be in %rax and can have the following values
    * (you can find these in syscalls.master) within the FreeBSD kernel source
-   * code):
+   * code).
+   * According to the type of fork and the flags, we make a decision about
+   * whether to COW on ghost memory or not, and save the decision in the flags
+   * filed of the current SVAThread. (1 is COW, and 0 is not COW.)
```

Comment:
Nitpick: you misspelled "field" as "filed."  FWIW, I make this mistake all the time (in fact, I made it and had to correct it while writing this comment.  :)